src/components/tabs.vue

```diff
@@ -89,7 +89,6 @@ export default class Tabs extends Mixins(CommonMixin) {
 
 .mobile-font {
   font-size: $px-14;
-  max-width: 120px;
```

Comment:
Fix for tab title:

![image](https://github.com/bcgov/namerequest/assets/10002889/4bec6265-d347-442c-b54d-61b455287eba)